### PR TITLE
Redesign of Streams page - includes video & customer logos

### PR DESCRIPTION
**Pull request by manjuapu**
@derrickdoo Please review.

### CODE REVIEW

**Comment by derrickdoo** [comment]
@manjuapu I'm running your branch locally and I noticed some layout/display issues that should be sorted out.

DESKTOP:
1. Clipping content on the right at certain browser widths:
image

2. Unnecessary padding appears to the left of the main content area at certain browser widths:
image

3. Horizontal nav should keep the same color scheme when you vertically scroll down the page. Can you just apply a white background with maybe 10px of padding-bottom to the nav to make it appear as if content is scrolling underneath the nav?
image

MOBILE:
1. Content is getting clipped on mobile
image

2. Video embed is getting clipped on mobile
image

3. "write your first app" cta should be at the top of the list
image

4. (not because of your changes) do you mind reducing the text size for the code examples for mobile
image

**Comment by manjuapu** [comment]
@derrickdoo  I have made changes as per your feedback, but need help with video clipping off.

**Comment by derrickdoo** [comment]
@manjuapu to fix the video clipping issue you need to switch over to relative sizing when your viewport gets smaller than the full size embed (525px) + the total left/right padding/margin (20px) on your content area. Here's the CSS you need:

`
@media only screen and (max-width: 545px) {
  .yt_series {
    width: 100%;
  }
}
`

There's also a few more tweaks you should make to get things synced up with the design comps.

1. Remove the default borders on your video embeds:

`
.yt_series{
  border: none;
}
` 

2. At 1125px wide, you're folding the video selector items down to just the 1-4 clickable bullets below the video embed. You should center the video and clickable bullets to be consistent with the rest of layout

`
@media only screen and (max-width: 1125px) {
    .yt_series {
        margin: 0 auto;
    }
    .video__list {
        text-align: center;
    }
}
`

Last thing just for readability I'd consider grouping all your media queries together. For instance you're looking for 

`
@media only screen and (max-width: 1125px)
`

In 4 places in style.css. Just group all that stuff up.

File: css/styles.css [hunk 1]
```diff
@@ -1031,50 +1054,438 @@ nav .btn {
...
+.navbar-fixed {
+    position: fixed;
+    background-color: #fff;
+    color: #fff;
+    margin-top: 0px;
+    padding: 10px 0px;
+    width: 92rem;
+    z-index: 1000;
+    top: 0px;
+    border-bottom: solid 1px #E5E5E5;
```
Comment (derrickdoo) [inline css/styles.css]: Feels fine without the border-bottom - please remove

File: css/styles.css [hunk 2]
```diff
@@ -1031,50 +1054,438 @@ nav .btn {
...
+    margin-bottom: 2rem;
+    text-align: center;
+    padding-top: 2rem;
+}
+.navbar-fixed {
+    position: fixed;
+    background-color: #fff;
+    color: #fff;
+    margin-top: 0px;
+    padding: 10px 0px;
```
Comment (derrickdoo) [inline css/styles.css]: padding: 10px 0 20px;

File: 0110/streams/index.html [hunk 3]
```diff
@@ -1,275 +1,311 @@
...
-    <div class="customer__cards">
-         <div class="customer__card">
-             <div class="customer__card__icon">
-            <img src="/{{version}}/images/icons/NYT.jpg" style="max-width:35%">
-          </div>  
-          <span class="customer__card__label">The New York Times uses Apache Kafka and the Kafka Streams API to store and distribute, in real-time, published content to the various applications and systems that make it available to the readers.</span>
-            <a href="https://www.confluent.io/blog/publishing-apache-kafka-new-york-times/">Learn More</a>
-             
+   <h1>Kafka Streams API</h1>
+   <ul class="streams-menu">
```
Comment (derrickdoo) [inline 0110/streams/index.html]: When this element doesn't completely fit in view, we need to make it so users can scroll over  to see the rest of the menu. Take a look at .nav-scroller when the viewport is below 650px wide. Lets just do the same thing here.

**Comment by manjuapu** [comment]
@derrickdoo I have made the changes. I was able to figure out the video responsiveness, coded in a slightly different way. Please review and let me know if I need to make any more changes.
Thanks!!

**Comment by furyus** [comment]
Please @ mention me when this gets merged - thanks for your hard work!

File: 0110/streams/index.html [hunk 4]
```diff
@@ -1,275 +1,311 @@
...
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
--->
-
-<script><!--#include virtual="../js/templateData.js" --></script>
-
+   Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (guozhangwang) [inline 0110/streams/index.html:2]: I need to point out that if we made a new release from 0.11.0 and do the copy-paste from `kafka` repo these changes will be auto-reverted as well. Do you want to submit a corresponding PR for `kafka` against 0.11.0 branch to keep the docs in-sync?

**Comment by guozhangwang** [comment]
Merged to asf-site.

**Pull request closed without merging**